concepts/numbers/about.md

```diff
@@ -19,6 +19,32 @@ There is no separate integer type in common everyday use.
 The `bigint` type is not designed to replace the `number` type for everyday uses.
 `42` is still a `Number`, not a `BigInt`.
 
+## Special Notations
+
+There are two types of notations:
+- Exponential notation(E-notation)
+- Underscore notation
+
+### Exponential notation
+The E-notation indicates a number that should be multiplied by 10 raised to a given power.
+The format of E-notation is to have a number, followed by `e` or `E`, than by the power of 10 to multiply by.
+```javascript
+num = 3.125e7; // Equals to 31250000
+// The notation essentially says, "Take 3.125 and multiply it by 10^7.
```

Comment:
* `const` is missing (we always try to provide code examples that follow the recommendations how you should usually write the code, even if it would work otherwise, also we always use `const` if `let` is not required because of a re-assignment)
* You can use `// =>` to demonstrate the result of an expression, like you would see it in a REPL.
* There is a quote missing at the end.

This is how it should look like, you can then apply it for other examples where needed.

```suggestion
const num = 3.125e7;
// => 31250000
// The notation essentially says, "Take 3.125 and multiply it by 10^7".
```